engine/player/unique_gear_shadowlands.cpp

```diff
@@ -2284,6 +2284,37 @@ void titanic_ocular_gland( special_effect_t& effect )
   } );
 }
 
+/**Ebonsoul Vise
+ * id=355327 damage debuff
+ * id=357785 crit pickup buff NYI
```

Comment:
Crit pickup stuff isn't implemented since I don't know how likely is that someone will model this in the APL. The orb that gives you the buff spawns at the location where target died.

I saw that melee version of this trinket has it implemented assuming the player would pick it up, but ranged would need some kind of rng roll on this I guess.

https://github.com/simulationcraft/simc/blob/shadowlands/engine/player/unique_gear_shadowlands.cpp#L2041-L2057

Also, ranged buff has different spell id than the melee on, and I can't seem to find it in the spell data 

![](https://i.imgur.com/MU178fx.png)